### PR TITLE
fix(tests): fix typo in test name

### PR DESCRIPTION
**Pull request by adriangb**
I'm only a casual convention commit message user, so please correct me if there's a better tag for something this trivial.

### CODE REVIEW

File: metadata-ingestion/tests/unit/test_schema_util.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def assert_field_paths_are_unique(fields: List[SchemaField]) -> None:
         assert len(avro_fields_paths) == len(set(avro_fields_paths))
 
 
-def assret_field_paths_match(
+def assert_field_paths_match(
```
Comment (shirshanka) [inline metadata-ingestion/tests/unit/test_schema_util.py:76]: Thanks for the contrib!
This function is used in a few places, so you will have to do a find/replace of this in all callers.

**Review by shirshanka — CHANGES_REQUESTED** [review]

File: metadata-ingestion/tests/unit/test_schema_util.py (hunk 1 above)
Comment (adriangb) [reply metadata-ingestion/tests/unit/test_schema_util.py:76]: Ah sorry my bad, this was a super drive by 1am edit on github.com, should have realized it was a util function not a `def test_...` function used once 🤦. Should be fixed in 65410437e.

Comment (shirshanka) [reply metadata-ingestion/tests/unit/test_schema_util.py:76]: thank you!

**Review by shirshanka — APPROVED** [review]
LGTM!

**Pull request merged**